mkdocs/docs/quick-start-demo.md

```diff
@@ -0,0 +1,183 @@
+
+    <div align="center">
+        <img width="500" src="../imgs/quick-start-demo/create-credential.png"></img>
+    </div>
+
+4. You can click on **View Link** on the next screen to check the generated QR code.
+
+    <div align="center">
+        <img width="500" src="../imgs/quick-start-demo/qr-code.png"></img>
+    </div>
+
+## Fetch the newly created credential
+Now we are back to the ID Holder role. They will use their mobile application to authenticate themselves by scanning the QR code generated by the issuer in the last step.
+
+<div align="center">
+    <img width="300" src="../imgs/quick-start-demo/quick-fetch-1.jpeg"></img>
+</div>
+
+Connect to the issuer:
+
+<div align="center">
+    <img width="300" src="../imgs/quick-start-demo/quick-fetch-2.jpeg"></img>
+</div>
+
+This will instantly trigger a notification on the mobile which will look like this:
+
+<div align="center">
+    <img width="300" src="../imgs/quick-start-demo/quick-notification.jpeg"></img>
+</div>
+
+Accept the credential:
+
+<div align="center">
+    <img width="300" src="../imgs/quick-start-demo/quick-fetch-4.jpeg"></img>
+</div>
+
+The ID Holder successfully retrieved the credential and it is visible on the app: 
+
+<div align="center">
+    <img width="300" src="../imgs/quick-start-demo/quick-fetch-6.jpeg"></img>
+</div>
+
+## Verify the ID holder credential
+Here comes the third role in this tutorial: the verifier. This could be represented by an organization that needs to check the accuracy of someone's credentials. In our use case, this organization wants to verify whether the ID holder actually attended our made-up Paris event.
+
+Here are the steps to verify the credential:
+
+1. Visit the [Verifier website](https://verifier-v2.polygonid.me/). As we are using a newly-created credential type, you need to choose **Custom** on the dropdown menu. Click on **Sign In**.
+
+    <div align="center">
+        <img width="600" src="../imgs/quick-start-demo/verifier-lp.png"></img>
+    </div>
+
+2. Now you will make use of the JSON-LD Context URL that was generated on the same process when you created the JSON schema URL in step 5 of [Create a Schema](#create-a-schema). Here is how the query should look like:
+
+    <div align="center">
+        <img width="600" src="../imgs/quick-start-demo/verifier-query.png"></img>
+    </div>
+
+3. After clicking on **Submit**, you should scan the resulting QR code and follow the instructions on the mobile app.
+
+    <div align="center">
+        <img width="300" src="../imgs/quick-start-demo/verifier-request.jpeg"></img>
+    </div>
+
+
+4. The process of generating the proof is then started:  
+
+    <div align="center">
+        <img width="300" src="../imgs/quick-start-demo/verifier-proof.jpeg"></img>
+    </div>
+
+5. And finally, the proof will be added to the Verifier environment.
```

Comment:
Probably makes more sense to say: "And finally, the proof will be validated by the Verifier environment."